spec/arrays.dd

```diff
@@ -43,26 +63,30 @@ int[3] s;
 ---------
 )
 
-        $(P These are analogous to C arrays. Static arrays are distinguished
-        by having a length fixed at compile time.
+        $(P Static arrays have a length fixed at compile time.
         )
 
         $(P The total size of a static array cannot exceed 16Mb.
```

Comment:
Isn't this a limitation imposed by the linker DMD uses and not necessarily a limitation of the language.

Reply:
The maximum length set set by Optlink for Win32. It is still a fixed size, however, on all platforms.